src/Pages/HomePage/Home.tsx

```diff
@@ -1,9 +1,11 @@
+import { Projects } from "./components/Projects/Projects";
 import { AboutUs } from './components/AboutUs/AboutUs';
 import { ClientsFeedback } from './components/ClientsFeedback/ClientsFeedback';
 import { ContactUs } from './components/ContactUs/ContactUs';
 import { Hero } from './components/Hero/Hero';
 import { TeamMember } from './components/TeamMember/TeamMember';
 
+      <Projects />
```

Comment:
what is it?...)

you have this line outside of the Home...

Reply:
Fixed